master-e2e-create-20190430 is broken

The error is:
```

time="2019-10-23T23:02:21Z" level=info msg="creating/updating cluster"
panic: interface conversion: interface {} is api.OpenShiftManagedClustersClient, not api.OpenShiftManagedClustersClient (types from different packages)
goroutine 1 [running]:
main.Client.createOrUpdatev20190430(0xc0003016c0, 0xc00056ab00, 0xc000196d50, 0xc00013bcc0, 0xc00057ce20, 0x15, 0xbe0ec0, 0xc0002e1220, 0xbe0740, 0xc0000be010, ...)
	/home/prow/go/src/github.com/openshift/openshift-azure/cmd/createorupdate/createorupdate.go:136 +0x5ff
main.Client.Create(0xc0003016c0, 0xc00056ab00, 0xc000196d50, 0xc00013bcc0, 0xc00057ce20, 0x15, 0xbe0ec0, 0xc0002e1220, 0xbe0740, 0xc0000be010, ...)
	/home/prow/go/src/github.com/openshift/openshift-azure/cmd/createorupdate/createorupdate.go:298 +0x62f
main.main()
	/home/prow/go/src/github.com/openshift/openshift-azure/cmd/createorupdate/createorupdate.go:248 +0xa5b
exit status 2 
```

--- a/cmd/createorupdate/createorupdate.go
+++ b/cmd/createorupdate/createorupdate.go
@@ -133,7 +133,7 @@ func (c Client) createOrUpdatev20190430(ctx context.Context) (*v20190430.OpenShi
 		return nil, fmt.Errorf("failed reading manifest: %v", err)
 	}
 	c.log.Info("creating/updating cluster")
-	resp, err := c.rpcs["2019-10-27-preview"].(v20190430client.OpenShiftManagedClustersClient).CreateOrUpdateAndWait(ctx, c.conf.ResourceGroup, c.conf.ResourceGroup, *oc)
+	resp, err := c.rpcs["2019-04-30"].(v20190430client.OpenShiftManagedClustersClient).CreateOrUpdateAndWait(ctx, c.conf.ResourceGroup, c.conf.ResourceGroup, *oc)
 	if err != nil {
 		return nil, err
 	}

